src/Analyzers/CSharp/Tests/ConstantInterpolatedString/ConstantInterpolatedStringTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.ConstantInterpolatedString;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Analyzers.UnitTests.ConstantInterpolatedString
+{
+    using VerifyCS = CSharpCodeFixVerifier<CSharpConstantInterpolatedStringAnalyzer, EmptyCodeFixProvider>;
+
+    public sealed class ConstantInterpolatedStringTests
+    {
+        // Should be C# 10 after release.
+        private static readonly LanguageVersion s_minimumSupportedVersion = LanguageVersion.Preview;
```

Comment:
This is now available as CSharp10 right?

Reply:
Yes. Fixed